cmd/operator-controller/main.go

```diff
@@ -52,18 +52,19 @@ import (
 
 	helmclient "github.com/operator-framework/helm-operator-plugins/pkg/client"
 
+	"github.com/operator-framework/operator-controller/internal/operator-controller/action"
+	"github.com/operator-framework/operator-controller/internal/operator-controller/applier"
+
 	ocv1 "github.com/operator-framework/operator-controller/api/v1"
 	catalogd "github.com/operator-framework/operator-controller/catalogd/api/v1"
-	"github.com/operator-framework/operator-controller/internal/action"
-	"github.com/operator-framework/operator-controller/internal/applier"
-	"github.com/operator-framework/operator-controller/internal/authentication"
-	"github.com/operator-framework/operator-controller/internal/catalogmetadata/cache"
-	catalogclient "github.com/operator-framework/operator-controller/internal/catalogmetadata/client"
 	"github.com/operator-framework/operator-controller/internal/contentmanager"
 	"github.com/operator-framework/operator-controller/internal/controllers"
 	"github.com/operator-framework/operator-controller/internal/features"
 	"github.com/operator-framework/operator-controller/internal/finalizers"
 	"github.com/operator-framework/operator-controller/internal/httputil"
+	"github.com/operator-framework/operator-controller/internal/operator-controller/authentication"
+	"github.com/operator-framework/operator-controller/internal/operator-controller/catalogmetadata/cache"
+	catalogclient "github.com/operator-framework/operator-controller/internal/operator-controller/catalogmetadata/client"
```

Comment:
I think all those code are valid for both.
@joelanford is doing a PR to consolidate the cache right

Reply:
We discussed it in the channel for now we can just move so all fine.
Closing as resolved.